x-pack/plugins/security_solution/public/entity_analytics/components/entity_store/components/missing_privileges_callout.tsx

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiCallOut, EuiCode, EuiText } from '@elastic/eui';
+import React from 'react';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { LineClamp } from '../../../../common/components/line_clamp';
+import type { EntityAnalyticsPrivileges } from '../../../../../common/api/entity_analytics';
+import { getAllMissingPrivileges } from '../../../../../common/entity_analytics/privileges';
+import { CommaSeparatedValues } from '../../../../detections/components/callouts/missing_privileges_callout/comma_separated_values';
+
+interface MissingPrivilegesCalloutProps {
+  privileges: EntityAnalyticsPrivileges;
+}
+
+export const MissingPrivilegesCallout = React.memo(
+  ({ privileges }: MissingPrivilegesCalloutProps) => {
+    const missingPrivileges = getAllMissingPrivileges(privileges);
+    const indexPrivileges = missingPrivileges.elasticsearch.index ?? {};
+    const clusterPrivileges = missingPrivileges.elasticsearch.cluster ?? {};
+    const featurePrivileges = missingPrivileges.kibana;
+    const id = `missing-entity-store-privileges`;
+    return (
+      <EuiCallOut
+        color="primary"
+        title={
+          <FormattedMessage
+            id="xpack.securitySolution.riskEngine.missingPrivilegesCallOut.title"
+            defaultMessage="Insufficient privileges to enable the Entity Store"
+          />
+        }
+        iconType={'iInCircle'}
+        data-test-subj={`callout-${id}`}
+        data-test-messages={`[${id}]`}
+      >
+        <LineClamp maxHeight="100%" lineClampHeight={4.4}>
```

Comment:
maybe extract these magic values?

Reply:
or is this component copy pasta?

Reply:
`maxHeight` is 100% because I don't wanna the container to have a scroll bar.
But `lineClampHeight` was trial and error. The `LineClamp` component doesn't work well with non-text content, so I used decimals to compensate.

I will add `lineClampHeight' to a constant and add comments.